LandMark/Feature/Home/Pin/Data/Location/LocationCoordinate.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LocationCoordinate.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 31/10/24.
+//
+
+import CoreLocation
+
+struct LocationCoordinate {
+    let latitude: Double
+    let longitude: Double
+    
+    init(latitude: Double, longitude: Double) {
+        self.latitude = latitude
+        self.longitude = longitude
+    }
+    
+    init(location: CLLocation) {
+        self.latitude = location.coordinate.latitude
+        self.longitude = location.coordinate.longitude
+    }
+}
```

Comment:
Também super nit-pick, mas imagino que o objetivo de `LocationCoordinate` existir sejá so pra abstrair o conceito de coordenadas independent do framework que usarmos, certo? Nesse caso, colocaria esse `init` em uma extensão separada (pode ser no mesmo arquivo mesmo). 

Só pra ficar bem clara a divisão.

```suggestion
}

// MARK: - LocationCoordinates + CoreLocation
extension LocationCoordinates {    
    init(location: CLLocation) {
        self.latitude = location.coordinate.latitude
        self.longitude = location.coordinate.longitude
    }
}
```

Reply:
Perfeito! Acho coerente!